.github/workflows/tests-dbstorage.yml

```
@@ -61,7 +65,7 @@ jobs:
     - name: Setup cache
       uses: actions/cache@v2
       env:
-        cache-name: test-rdbstorage
+        cache-name: test-dbstorage
```

Since Optuna doesn't have `DBStorage`, how about putting emphasis on `test with server`? 
```suggestion
        cache-name: test-storage-with-server
```

---

Ok, I will fix it.

---

Thank you!

---

Fixed.

---

Thanks!